Scripts/Python/grsnNexusBookMachine.py

```diff
@@ -202,6 +228,7 @@ def OnNotify(self,state,id,events):
             PtDebugPrint("grsnNexusBookMachine.OnNotify:\tadding you to private chat channel %d" % (chatChannel.value),level=kDebugDumpLevel)
             PtSendPrivateChatList(self.GetPlayersInChatDistance())
             PtSendKIMessageInt(kSetPrivateChatChannel, chatChannel.value)
+            AreWeInRoom = True
```

Comment:
This looks like it's vulnerable to problems if someone is running modified Python. I think the initial if statement needs to also include a check against `PtFindAvatar(events) == PtGetLocalAvatar()` to ensure we only fire for our own notifications. The cutscene camera is vulnerable to the same problem.